test/runbld/main_test.clj

```diff
@@ -130,7 +130,7 @@
                          :process
                          :exit-code)))
             (is (empty? @email))
-            (is (.contains (slack-msg) "SUCCESS"))))))))
+            (is (not (empty? (slack-msg))))))))))
```

Comment:
So, I'm guessing the success case no longer contains any predetermined string you can check for?